PySDM/physics/isotope_kinetic_fractionation_factors/jouzel_and_merlivat_1984.py

```diff
@@ -1,6 +1,9 @@
 """
-kinetic fractionation factor from [Jouzel & Merlivat 1984](https://doi.org/10.1029/JD089iD07p11749)
-(as eq. 3e for n=1 in [Stewart 1975](https://doi.org/10.1029/JC080i009p01133))
+kinetic fractionation factor  for liquid and ice from
```

Comment:
the kinetics described here concern diffusion of vapour in air, adding "liquid and ice" seems misleadi g as there is nothing related to solid or liquid phase here